config/application.yml.example

```diff
@@ -70,6 +71,7 @@ production:
   otp_delivery_blocklist_findtime: '5'
   otp_delivery_blocklist_maxretry: '5'
   participate_in_dap: 'no'
+  password_pepper: 'rake secret'
```

Comment:
By "manually", we mean that we have to SSH into the servers and update `config/application.yml` manually. Justin is working on automating this. In save-ferris, aka ICAM, we did this with Chef and stored the secrets in an encrypted data bag.